src/test/java/org/cactoos/proc/ForEachTest.java

```diff
@@ -60,4 +63,24 @@ void testProcIterable() throws Exception {
         ).affirm();
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    void worksWithGenerics() throws Exception {
+        final List<? super Number> list = new LinkedList<>();
+        new ForEach<>(
+            (Proc<? super Number>) list::add
+        ).exec(
+            new IterableOf<>(
+                Integer.valueOf("1"),
+                Double.valueOf("2"),
+                Long.valueOf("3")
```

Comment:
@andreoss those 3 things are kind of strange… why not just use 1, 1d and 1L for example? or if you want to be sure to have an object and not a primitive, new Double(), new Integer(), etc?

Reply:
@victornoel I think it was a way to avoid triggering `MagicNumber` check. Removed, it was unnecessary.